Jenkinsfile

```diff
@@ -30,11 +30,7 @@ pipeline {
         }
         // Use the pomfix tool to validate that bundle dependencies are properly declared
         stage('Validate Poms') {
-            agent { label 'linux-small' }
             steps {
-                retry(3) {
```

Comment:
@mcalcote are we losing the retry functionality by switching away from the scripted syntax? Is there a way we can advise this still?

Reply:
Looking through the documentation about it, I didn't see the same options for the functionality we previously had set. https://jenkins.io/doc/book/pipeline/syntax/#options
One advantage to this change is that we significantly reduce the number of checkouts we are performing overall. I feel like, in this case, the benefit would outweigh the loss of retry on a per checkout basis.

Reply:
It looks like `retry` should still be a "basic pipeline step" (https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#code-retry-code-retry-the-body-up-to-n-times). So, I'm not actually sure that we're unable to just leave that retry block in there. https://github.com/jenkinsci/pipeline-model-definition-plugin/wiki/Running-multiple-steps#timeouts-retries-and-more

Reply:
yeah we can use the `retry` step with `checkout` but we'll need to add the `skipDefaultCheckout() ` option to the pipeline to avoid automatically running a checkout as part of the `agent` setup

Reply:
I can add the option and set the checkout with retry on the first stage that uses the `linux-large` agent.